tests/test_FBbasis2D.py

```diff
@@ -134,3 +135,32 @@ def testComplexCoversionErrorsToReal(self, basis):
 
         # Try a 0d vector, should not crash.
         _ = basis.to_real(cv1.reshape(-1))
+
+
+params = [pytest.param(256, np.float32, marks=pytest.mark.expensive)]
+
+
+@pytest.mark.parametrize(
+    "L, dtype",
+    params,
+)
+def testHighResFBBasis2D(L, dtype):
+    seed = 42
+    basis = FBBasis2D(L, dtype=dtype)
+    sim = Simulation(
+        n=1,
+        L=L,
+        dtype=dtype,
+        amplitudes=1,
+        offsets=0,
+        seed=seed,
+    )
+    im = sim.images[0]
+
+    # Round trip
+    coeff = basis.evaluate_t(im)
```

Comment:
Shouldn't we be testing `expand`?

Reply:
Ah yes, thanks. That tightens up the tolerance quite a bit. Done.